shared/fixes/bash/audit_rules_kernel_module_loading_rmmod.sh

```diff
@@ -5,5 +5,5 @@
 
 # Perform the remediation for both possible tools: 'auditctl' and 'augenrules'
 
-fix_audit_watch_rule "auditctl" "/usr/sbin/rmmod" "x" "modules"
-fix_audit_watch_rule "augenrules" "/usr/sbin/rmmod" "x" "modules"
+fix_audit_watch_rule "auditctl" "/sbin/rmmod" "x -F auid!=4294967295" "module-change"
+fix_audit_watch_rule "augenrules" "/sbin/rmmod" "x -F auid!=4294967295" "module-change"
```

Comment:
Seems like both should be audited, actually.

Reply:
The XCCDF and OVAL should be updated too. Is this something you could do, @chrisle925 ?